### DISCUSSION TITLE
application refused

### DISCUSSION THREAD

**Discussion by conquer1the2world3** [community/community#48523] [GitHub Education]
I have submitted my student affirmation so many times, my dated school id, my dated course schdule, my dated enrollment letter.
All of information as much as I reminded. But i just got autoreply emails one by one said need more information.
Please let me know wgat else information do I need to submit ok?
I really wang to get the student pack to study coding with servers.
Thanks!

**Comment by slano-ls** [comment]
I did the same thing and had a lot of issues, in the end I took a different picture made sure it was as clear as possible; in my scenario I had to flip my image horizontally in order for it to the scanner to be able to approve it. Keep trying out different images!

**Reply by conquer1the2world3** [reply]
Thanks for for your reply.
I try to make a almost 10MB pitcture and flip it horizontally to upload.
But it seems that it doesn't have a effect.

**Comment by samus-aran** [comment]
Hi @conquer1the2world3  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍
